learning/tour-of-beam/frontend/pubspec.lock

```diff
@@ -390,6 +402,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.7.0"
+  hive:
```

Comment:
This was brought by https://pub.dev/packages/autotrie
and will be tree-shaken away.